system_reports/osx-xcode-12.5.x.log

```diff
@@ -1 +1,1230 @@
-[TBD]
\ No newline at end of file
+ (i) Ignore Errors: enabled
+
+#
+# This System Report was generated by: https://github.com/bitrise-io/osx-box-bootstrap/blob/master/system_report.sh
+#  Pull Requests are welcome!
+#
+
+
+=== Revision / ID ======================
+* BITRISE_OSX_STACK_REV_ID: v2021_01_08
+========================================
+
+
+=== Pre-installed tool versions ========
+* Go: go version go1.15.5 darwin/amd64
+* Python: Python 2.7.16
+* Node.js: v12.20.0
+* NPM: 6.14.8
+* Yarn: 1.22.10
+* Java: openjdk version "1.8.0_282"
+OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_282-b08)
+OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.282-b08, mixed mode)
+* Flutter: Flutter 1.22.1 • channel stable • https://github.com/flutter/flutter.git
+
+* git: git version 2.29.2
+* git lfs: git-lfs/2.13.1 (GitHub; darwin amd64; go 1.15.5)
+* mercurial/hg: Mercurial Distributed SCM (version 5.6.1)
+* curl: curl 7.64.1 (x86_64-apple-darwin20.0) libcurl/7.64.1 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.41.0
+* wget: GNU Wget 1.20.3 built on darwin20.1.0.
+* rsync: rsync  version 2.6.9  protocol version 29
+* unzip: UnZip 6.00 of 20 April 2009, by Info-ZIP.  Maintained by C. Spieler.  Send
+* zip: This is Zip 3.0 (July 5th 2008), by Info-ZIP.
+* tar: bsdtar 3.3.2 - libarchive 3.3.2 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.6
+* tree: tree v1.8.0 (c) 1996 - 2018 by Steve Baker, Thomas Moore, Francesc Rocher, Florian Sesser, Kyosuke Tokoro 
+
+* brew: Homebrew 2.7.7
+Homebrew/homebrew-core (git revision 377d7b; last commit 2020-12-16)
+Homebrew/homebrew-cask (git revision f606d; last commit 2020-12-16)
+* Ansible: ansible 2.10.3
+  config file = /Users/vagrant/git/ansible.cfg
+  configured module search path = ['/Users/vagrant/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
+  ansible python module location = /usr/local/Cellar/ansible/2.10.4/libexec/lib/python3.9/site-packages/ansible
+  executable location = /usr/local/bin/ansible
+* gtimeout: timeout (GNU coreutils) 8.32
+* watchman: 4.9.0
+* flow: Flow, a static type checker for JavaScript, version 0.138.0
+* carthage: 0.36.0
```

Comment:
can you update carthage version to latest one 0.37.0

Reply:
We will, we can't actually release it yet unfortunately, but when we do we will make sure the have carthage updated